packages/lib/modules/pool/pool.helpers.ts

```diff
@@ -302,7 +303,12 @@ const shouldBlockV3PoolAdds = false
  * Returns true if we should block the user from adding liquidity to the pool.
  * @see https://github.com/balancer/frontend-v3/issues/613#issuecomment-2149443249
  */
-export function shouldBlockAddLiquidity(pool: Pool) {
+export function shouldBlockAddLiquidity(pool: Pool, poolMetadata?: PoolMetadata) {
```

Comment:
Working on https://github.com/balancer/frontend-monorepo/issues/702 and that is related to this one and have a question: should we show tooltip reasons (getPoolAddBlockedReason) even when it is forced to be allowed by the metadata?

Reply:
In instances where it is explicitly allowed, I would not show the reason, it should be covered by the custom pool warnings